shell/platform/embedder/embedder.cc

```diff
@@ -1388,6 +1388,14 @@ FlutterEngineResult FlutterEngineSendWindowMetricsEvent(
   metrics.physical_width = SAFE_ACCESS(flutter_metrics, width, 0.0);
   metrics.physical_height = SAFE_ACCESS(flutter_metrics, height, 0.0);
   metrics.device_pixel_ratio = SAFE_ACCESS(flutter_metrics, pixel_ratio, 1.0);
+  metrics.physical_view_inset_top =
```

Comment:
I realize this is just plumbing things through, but, you could add a test for this in the `embedder_unittests` harness. Any of the tests in `embedder_unittests.cc` could be a fine starting point.

Reply:
Done.